### ISSUE TITLE
Why create this, without any issues or PRs to enzyme itself?

### ISSUE THREAD

**Issue by ljharb** [commercetools/enzyme-extensions#8]
While `until` has been under consideration since long before enzyme 3, I don't recall any issues asking for any of the features in this repo.

Is there any particular reason there was no attempt to add these features to enzyme proper?

**Comment by tdeekens** [comment]
> Is there any particular reason there was no attempt to add these features to enzyme proper?

Not really. We wanted to ensure that the solutions work for us, then see how valuable it is to others and then propose them as an addition to enzyme itself. If you have interest in us adding this to enzyme itself we're more than happy to help out.

**Comment by emmenko** [comment]
We're happy to contribute to enzyme of course, if you think this might be a consider. Thanks for reaching out to us! 🙌

**Comment by ljharb** [comment]
I think each one should be evaluated independently, but if you’ve found them useful, it’s always helpful to discuss the use cases. A package like this shouldn’t be necessary - enzyme should be able to meet the needs of its users on its own.

**Comment by dferber90** [comment]
I now opened https://github.com/airbnb/enzyme/pull/1863 which enhances `enzyme` with `renderProp`, the most useful method of this package.

We're currently using `renderProp` ~320 times in our tests. The other methods (`drill` and `until`) didn't really gain adoption internally,  so I did not open PRs to `enzyme` for them.

With that, this issue seems ready to be closed. Thanks again for reaching out! I'd be excited if this makes it into `enzyme`!
